stdlib/2and3/genericpath.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import List
+import sys
+
+def exists(path: str) -> bool: ...
+def isfile(path: str) -> bool: ...
+def isdir(s: str) -> bool: ...
+def getsize(filename: str) -> int: ...
+def getmtime(filename: str) -> float: ...
+def getatime(filename: str) -> float: ...
+def getctime(filename: str) -> float: ...
+def commonprefix(m: List[str]) -> str: ...
+
+if sys.version_info[0] >= 3.4:
```

Comment:
That's not how you check versions.

Reply:
Yeah... I don't really know what I was thinking there. Apologies.